homework_7_261122/src/products/Product.java

```diff
@@ -0,0 +1,37 @@
+package products;
+
+import java.sql.SQLOutput;
```

Comment:
remove unused import

Reply:
Fixed.

Reply:
I wonder where it might have come from? (import java.sql.SQLOutput;)

Reply:
Who knows:)